docs/rules.md

```diff
@@ -498,12 +482,22 @@ Location header must be supported for all async operations that return 202.
 
 Please refer to [lro-location-header.md](./lro-location-header.md) for details.
 
+### ProvisioningStateSpecifiedForLROPatch
+
+This is a rule introduced to validate if a LRO PATCH operations response schema has "ProvisioningState" property specified for the 200 status code.
+
+Please refer to [lro-patch-provisioning-state-specified.md](./lro-patch-provisioning-state-specified.md) for details.
+
 ### LroPatch202
 
 Async PATCH should return 202.
 
 Please refer to [lro-patch202.md](./lro-patch202.md) for details.
 
+For long running (LRO) post operations, 'long-running-operation-options' must be present and have the 'final-state-via' property set to 'azure-async-operation'.
```

Comment:
>final-state-via [](http://example.com/codeflow?start=103&length=15)

lets remove all rules related to final-state-via. Earlier there was a misunderstanding about what the final-state-via property was meant for. We no longer need this enforcement.

Reply:
Made an ADO item for this.